### PR TITLE
Pull out cache controller to be reusable

### PR DESCRIPTION
**Pull request by cstamas**
As same logic will be needed by Maven2 group merge caching logic to control cached merge results.

CI
http://bamboo.s/browse/NX3-OSSF578

### CODE REVIEW

File: components/nexus-repository/src/main/java/org/sonatype/nexus/repository/cache/CacheController.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.repository.cache;
+
+import java.io.IOException;
+
+import javax.annotation.Nullable;
+
+import org.sonatype.nexus.repository.view.Content;
```
Comment (sonatype-zion) [inline components/nexus-repository/src/main/java/org/sonatype/nexus/repository/cache/CacheController.java]: MINOR Remove this unused import 'org.sonatype.nexus.repository.view.Content'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by sonatype-zion** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by mcculls** [comment]
+1

**Comment by cstamas** [comment]
http://bamboo.s/browse/NX3-OSSF578-2 :+1:

**Comment by cstamas** [comment]
a1983b011c9c115f7c775f27a6bcd4c2e3a6d1ba

**Pull request closed without merging**